### PR TITLE
Creaste user helper function that creates ideas

### PR DESCRIPTION
**Pull request by VladimirKarin**
https://vladimirkarin.atlassian.net/browse/FR-25

### CODE REVIEW

File: modals/ideas.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const { v4 } = require('uuid');
+
+const { getIdeas, setIdeas } = require('./utils/storage.js');
```
Comment (fedosejev) [inline modals/ideas.js:3]: Do you use these functions anywhere in this file?

**Comment by fedosejev** [comment]
Why did you name the directory `modals`?

File: modals/ideas.js (hunk 1 above)
Comment (VladimirKarin) [reply modals/ideas.js]: Not in this function. you're right. 
Regarding the name of directory, I remember, that we named it the same way in the prev. project.

Comment (fedosejev) [reply modals/ideas.js]: The name `modals` makes no sense here. Did you mean `models`?

Comment (VladimirKarin) [reply modals/ideas.js]: Yes, indeed. Fixed.

**Review by fedosejev — APPROVED** [review]

**Pull request merged**